sw/device/lib/testing/kmac/kmac_vector_generate.c

```diff
@@ -0,0 +1,67 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "KeccakHash.h"
```

Comment:
Per the OpenTitan style guide, includes should generally be rooted at the repository root. https://docs.opentitan.org/doc/sg/c_cpp_coding_style/#included-files

I admit this is a special circumstance because it's including a header from outside the OpenTitan repo, but I'd still try to use the full(er) path, `lib/high/Keccak/FIPS202/KeccakHash.h` and cut down on the `includes` in `third_party/xkcp/BUILD.xkcp.bazel`.

Reply:
An example of how you would reference a header in a Bazel vendored external library is shown [here](https://github.com/lowRISC/opentitan/blob/aaad9d3fe4d8d66b2d84525d44f06f2b75543ec9/sw/device/lib/testing/test_framework/freertos_hooks.c#L6). I believe this would be `#include external/xkcp/lib/high/Keccak/FIPS202/KeccakHash.h`, implementing the pattern: `#include external/<bazel repo name>/path/to/header`.

To clarify, I don't think using the full include path will necessarily cut down ont he `includes` in `third_party/xkcp/BUILD.xkcp.bazel`, as those are needed to compile the `cc_libraries` in an external repository that may not follow the same code style guide, i.e., src files in the vendored in code might not use full include paths. For example, [it does not look like the the XKCP codebase uses full include paths.](https://github.com/XKCP/XKCP/blob/64404beeeb261b08a1076fe2f076e4e28dd9b040/lib/high/Keccak/SP800-185/SP800-185.c#L18)

Reply:
I also thought about patching the header files of XKCP (so that they are using full paths), but then I thought maybe it a bit too much. I will make sure to use the full path for OT side code though. Thanks.

Reply:
Resolving since this change is no longer a part of this PR.